Translation correction (#256)
07th-mod/higurashi-console-arcs: Update/yoig_005_choice1.txt

			NULL, "\"It'll be fine.", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s10/00/410100784", 256, TRUE);
	OutputLine(NULL, "換気するのを待つくらい、どうってことねぇだろ？",

> 			NULL, " All we've got to do is wait for it to ventilate.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " All we need to do is wait for it to ventilate.", Line_WaitForInput);
```

//rなるほど…。ひっかけるつもりで窓ガラスを割ったことは伏せておいたが、…やはり、確定か。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "なるほど…。ひっかけるつもりで窓ガラスを割ったことは伏せておいたが、…やはり、確定か。",

> 			NULL, "I see... I was planning to bluff him by hiding the fact we broke the windows... but seems I was right on the money.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I see... I was planning to bluff him by hiding the fact that we broke the windows... but seems I was right on the money.", Line_Normal);
```

			NULL, "\"Oh, yeah. That's right.", Line_WaitForInput);
	ModPlayVoiceLS(4, 41, "ps3/s10/41/410200352", 256, TRUE);
	OutputLine(NULL, "…荒川さんは売れっ子ライターなんですよね。",

> 			NULL, " ...And you're a popular writer, Arakawa-san, right?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " ...And you're a popular writer, right, Arakawa-san?", Line_WaitForInput);
```

//r大学のことは、あまり話したくないらしい。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "大学のことは、あまり話したくないらしい。",

> 			NULL, "He didn't seem like he wanted to discussing his college all that much.", Line_Normal);

Reviewer: ```suggestion
			NULL, "He didn't seem all that interested in discussing his college", Line_Normal);
```
